filter.js

```diff
@@ -228,7 +237,10 @@ const sentEventClick = (eventName) => {
         if (foundEvent) {
           sentEventClick(foundEvent.title);
         }
+      } else if (event.target.classList.contains("tagsFilter")) {
```

Comment:
raczej nie `else` tylko zwykly if, gdyby się tak przydarzyło że miałbyś obiekt o dwóch klasach to nie zadziała tak jakbyć chciało . 

ofc tu jest ok :D